tests/Backend/Snowflake/ImportTypedTableTest.php

```diff
@@ -128,7 +128,6 @@ public function testImportEmptyValues(string $loadFile, array $expectedPreview):
 
         $payload = [
             'name' => 'with-empty',
-            'primaryKeysNames' => [],
```

Comment:
Tady jsi to schválně upravit v jiném testu? (jakože přijde mi to OK, nemyslím si, že bysme neměli otestovat i variantu kdy tam pošleš prázdné pole)

Reply:
trochu nechapem, ale je tam kopec inych testov kde sa tam posiela, prave to prazdne pole. Ja som to naschval upravil len v kazdej triede kde som to nasiel